File: CS/CodeBehind/EFCore/InstantFeedbackMode/Issues/IssuesContext.cs
using Microsoft.EntityFrameworkCore;

namespace EFCoreIssues.Issues {
    public class IssuesContext : DbContext {
        static readonly DbContextOptions<IssuesContext> options = new DbContextOptionsBuilder<IssuesContext>()
           .UseInMemoryDatabase(databaseName: "Test")
           .Options;
        public IssuesContext()
            : base(options) {
        }

        public DbSet<Issue> Issues { get; set; }
        public DbSet<User> Users { get; set; }
    }
}

File: VB/ViewModel/EFCore/InstantFeedbackMode/Issues/IssuesContext.vb
Imports Microsoft.EntityFrameworkCore

Namespace Issues
    Public Class IssuesContext
        Inherits DbContext

        Private Shared ReadOnly options As DbContextOptions(Of IssuesContext) = New DbContextOptionsBuilder(Of IssuesContext)().UseInMemoryDatabase(databaseName:="Test").Options

        Public Sub New()
            MyBase.New(options)
        End Sub

        Public Property Issues As DbSet(Of Issue)
        Public Property Users As DbSet(Of User)
    End Class
End Namespace
